docs/TheoryDeepPixels.rst

```diff
@@ -0,0 +1,491 @@
+======================
```

Comment:
This renders properly so it's probably fine, but the other .rst sections are formatted with underlined #:

Theory of Deep Samples
####################